engine/util/src/main/scala/pl/touk/nussknacker/engine/util/config/ScalaBinaryConfig.scala

```diff
@@ -0,0 +1,17 @@
+package pl.touk.nussknacker.engine.util.config
+
+import java.util.Collections
+
+import com.typesafe.config.{Config, ConfigFactory}
+
+object ScalaBinaryConfig {
+
+  val scalaBinaryVersion: String = util.Properties.versionNumberString.replaceAll("(\\d+\\.\\d+)\\..*$", "$1")
```

Comment:
In `Properties.versionNumberString` we have nk version? If so, can you change name? Because we now we interested in at least four versions: nk ui version, nk runtime extension version, scala version, flink version. If u can't at least extract local variable `nkVersion`. Also can we have in properties scala version so it regex won't be needed?

Reply:
no, in Properties.versionNumberString there is exact scala library version.

Reply:
Ok, it is `scala/util/Properties`... What do u thing about renaming this method to scalaMajorVersion? I was confused because I thought that you here extract scala version from some other version... because eg. 2.12.3 it is version of scala binary.

Reply:
oook, by scalaBinary I meant binary compatilibility - I'll change to major :+1:

Reply:
done, changed to major everywhere